src/applications/edu-benefits/tests/0993/00.0093-opt-out.e2e.spec.js

```diff
@@ -55,5 +54,3 @@ module.exports = E2eHelpers.createE2eTest(client => {
   client.axeCheck('.main');
   client.end();
 });
-
-module.exports['@disabled'] = !manifest.production;
```

Comment:
This is interesting. How would we ensure this isn't being used at runtime?

Reply:
I'm not sure I understand the question.